### ISSUE TITLE
wiki/tutorial issue - Making a New Learning Environment - Very low framerate

### ISSUE THREAD

**Issue by Dyonisian** [Unity-Technologies/ml-agents#1092]
Hi I'm not sure if this is the write place to leave this issue but it's still useful feedback.
The wiki tutorial for "Making a New Learning Environment" has some issue. I'm guessing some of the settings aren't explicitly mentioned which leads to terrible lag during the training process. I copied settings from one of the samples instead. Its either the academy settings (quality, timescale) or the agent settings (decision frequency). With the default settings the frame rate is extremely low. It would be helpful to have some documentation for proper settings, optimization, and how to prevent lag. 

Also, it would be helpful to mention in the initial "setting up the project" section that the .net framework has to be changed and the ENABLE_TENSORFLOW flag has to be set. It's left out which is confusing, though the other getting start tutorial teaches this.

Thanks.

**Comment by awjuliani** [comment]
Hi @Dyonisian,

Thanks for pointing these issues out. Would you mind mentioning the specific changes which led to smoother training performance?

**Comment by Dyonisian** [comment]
Hi @awjuliani, I think it was specifically the quality and timescale settings. The relationship between target FPS and timescale isn't clear - will the timescale be limited automatically if target FPS isn't met? The default values have timescale at 1 I think, and quality at the highest possible. Target FPS is -1. I changed that to 60 too. I also changed decision frequency to 5 from the default 1, but later when I switched it back to 1 it was still fine, so that wasn't the issue. However in my test a decision frequency of 5 gave much better training results than 1 did for some reason. If this is a consistent observation then it could be mentioned somewhere too.
